### PR TITLE
move cookies to graveyard

### LINKED ISSUE #7810: Remove cookies module from stdlib
This module really shouldn't be in the stdlib in the first place IMO

### PR DESCRIPTION
**Pull request by narimiran**
closes #7810 

[graveyard PR](https://github.com/nim-lang/graveyard/pull/5)

### CODE REVIEW

**Comment by dom96** [comment]
"../lib/pure/cgi.nim(32, 31) Error: cannot open file: cookies"

**Comment by Araq** [comment]
``cgi.nim`` should also be moved to the graveyard and it's ``escapeUrl`` or whatever is used from there moved to ``uri.nim``.

**Comment by narimiran** [comment]
> cgi.nim should also be moved to the graveyard and it's escapeUrl or whatever is used from there moved to uri.nim.

I've tried to remove `cgi.nim`, but locally when I try to *run* two tests (`tests/gc/growobjcrash.nim` and `tests/deps/jester-#head/jester/request.nim`) I have `Error: undeclared identifier: 'decodeData'`.  
Testing with `koch tests cat deps` and `koch tests cat gc` passes.

Let's wait and see what Travis CI says.

---

I don't know if this is the way to remove `cgi` (by adding whatever is needed to `uri`), please take a look and advise.

File: tests/deps/jester-#head/jester/private/utils.nim [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Copyright (C) 2012 Dominik Picheta
 # MIT License - Look at license.txt for details.
-import parseutils, strtabs, strutils, tables, net, mimetypes, asyncdispatch, os
-from cgi import decodeUrl
+import parseutils, strtabs, strutils, tables, net, mimetypes, asyncdispatch, os, uri
```
Comment (dom96) [inline tests/deps/jester-#head/jester/private/utils.nim]: Errr, wtf? Why is Jester in Nim's source tree?

Comment (Araq) [reply tests/deps/jester-#head/jester/private/utils.nim]: Because so that tests are self-contained without Nimble dependencies.

Comment (dom96) [reply tests/deps/jester-#head/jester/private/utils.nim]: You can install a specific commit hash of Jester using Nimble, why not do that? or use a git submodule?

Comment (Araq) [reply tests/deps/jester-#head/jester/private/utils.nim]: We used to do that but there was a period in the transition to 0.19 where Nimble would just fail. It also made local testing harder, I don't run these travis/appveyor scripts on my local machines.

Comment (dom96) [reply tests/deps/jester-#head/jester/private/utils.nim]: So don't put the Nimble commands into the travis script :/

**Pull request closed without merging**